pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -4170,9 +4169,9 @@ def _test_offset(self, offset_name, offset_n, tstart, expected_utc_offset):
 
     def _make_timestamp(self, string, hrs_offset, tz):
         if hrs_offset >= 0:
-            offset_string = "{hrs:02d}00".format(hrs=hrs_offset)
+            offset_string = f"{hrs_offset:02}00"
```

Comment:
Is the `d` format specifier automatically inferred based on supplied object here? Might be worth double checking this modification

Reply:
No it doesn't automatically infer the object unfortunately.

Reply:
Sure that or I think you can just keep the same `02d` format spec